client/src/components/MovieOverview/MovieOverview.spec.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react'
+import MovieOverview from './MovieOverview'
+
+describe('MovieOverview', () => {
+  it('renders', () => {
+    render(<MovieOverview />)
+
+    const h2Element = screen.getByRole('heading', { level: 2 })
+    expect(h2Element).toBeInTheDocument()
+
+    const movieCountText = screen.getByText(/\Moviecount:/)
+    expect(movieCountText).toBeInTheDocument()
```

Comment:
Does the test make actual get requests? 

In an ideal world we would mock axios/fetch and check if they were called

Reply:
I went with what is/should be displayed at least on the page. The 'Moviecount: ' is static. 

Right now it doesnt render any content that is done from the get request. Changed line 12 to '.not' and got the following message:
`expected document not to contain element, found <p class="MovieOverview__MovieCount-sc-103iaea-1 iJrHfT">Moviecount: 0</p> instead`